### ISSUE TITLE
Remove permission nodes from claim /trust?

### ISSUE THREAD

**Issue by RoboMWM** [GriefPrevention/GriefPrevention#2326]
remove handling of permission.node syntax for /trust and /acb

I had something written up about this somewhere, but especially so for acb: permissions only apply while player is online; player will appear to have less/negative claimblocks without any reliable way to indicate otherwise. Affects /claimslist and claim expiration, as well as the "auto fix negative claimblocks" feature.

The other reasons simplify command handling, database storage, and claim permission-checking

**Comment by Jikoo** [comment]
Do you mean trusting a permission via square brackets or something? Not aware of GP having any commands that aren't in the command list or anything.

**Comment by RoboMWM** [comment]
Yea. I think this used to be an issue with geyser, but there were other reasons why I wanted to remove this (one of them being that people expected the permissions to work offline, etc.) Though I'm not sure how big of an impact it would be to remove this.

**Comment by Jikoo** [comment]
I don't think it would be a big deal to remove, but we could also enforce the `[permission.node]` format that GP internally uses for permissions to help avoid `.name` being an issue. Pretty sure the only places the permission format is handled is when the command is run and when ClaimPermission is checked.

**Comment by RoboMWM** [comment]
well I know it was definitely used for /acb commands, not just /trust.

**Comment by RoboMWM** [comment]
I'll likely remove from /acb. Not sure of the use cases for these to be on claims? I can imagine some niche cases, like team or group/clan-based claims, but I'd imagine those are already being managed by another plugin...?

**Comment by Jikoo** [comment]
Re: acb: Only use case really is permission-based claim increases, which I believe generally would be better left to a plugin like GPAccruals.

Re: claims: Definitely niche, not aware of an addon that would replace the functionality though. I used it rarely for claims where various lower ranking staff were enlisted to help with prepping projects and I didn't want to have to type 20 names.
